src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -833,6 +833,11 @@ public static FormulaValue EncodeHTML(IRContext irContext, StringValue[] args)
             return new StringValue(irContext, encoded);
         }
 
+        public static FormulaValue PatchRecord(IRContext irContext, FormulaValue[] args)
+        {
+            return MutationUtils.MergeRecords(args).ToFormulaValue();
+        }
+

```

Comment:
This shouldn't be in LibraryText. LibraryTable is not ideal (this is only for records, after all), but is better than text.

Reply:
Thanks for catching this. I probably clicked over the wrong file since I was aiming for LibraryTable originally. The fact they are next to each other validates my excuse! 😁